### PR TITLE
feat: Add single cluster network connectivity visual analysis

### LINKED ISSUE #127: Add network connectivity visual analysis
**What would you like to be added**:
We need a deeper analysis of network problems, not just the results of connectivity verification. I think providing these functions can support targeted analysis of network problems. 
It can display the analysis results of network faults very intuitively, maybe introducing kosmosctl would be a good choice.

**Why is this needed**:
When I finished checking network connectivity, the command like: "kosmosctl dr -r ghcr.io/kosmos-io --src-kubeconfig A-kubeconfig --dst-kubeconfig B-kubeconfig". 
image
Some nodes are not connected to the network. At this time, I want to further investigate what caused the network connection failure. For example, a certain configuration error occurred. I found that I did not have a good way to complete it through Kosmos.

### PR DESCRIPTION
**Pull request by ONE7live**
#### What type of PR is this?

**feat:** Add single cluster network connectivity visual analysis
#### What does this PR do?

Cluster network visual analysis, add single cluster network configuration visual analysis command, e.g: 
kosmosctl analysis cluster --name=foo --kubeconfig=~/kubeconfig/config
#### Which issue(s) does this PR fix?

Fixes #127
#### Special notes for your reviewer:

/cc @wuyingjun-lucky 
#### Does this PR introduce a user-facing change?

```release-note

```

### CODE REVIEW

File: pkg/clusterlink/network/interface.go [hunk 1]
```diff
@@ -42,3 +42,9 @@ func NewNetWork() NetWork {
 	dn.InitSys()
 	return dn
 }
+
+func NewAnalysisNetwork() NetWork {
+	dn := &DefaultNetWork{}
```
Comment (wuyingjun-lucky) [inline pkg/clusterlink/network/interface.go]: recommend :
 implements : network interface Network

Comment (ONE7live) [reply pkg/clusterlink/network/interface.go]: ok, I've modified and add tag because for network analysis there is no need to perform 'initSys' logic again

**Pull request merged**